test/parallel/test-runner-run-coverage.mjs

```diff
@@ -157,7 +157,8 @@ describe('require(\'node:test\').run coverage settings', { concurrency: true },
       const thresholdErrors = [];
       const originalExitCode = process.exitCode;
       assert.notStrictEqual(originalExitCode, 1);
-      const stream = run({ files, coverage: true, lineCoverage: 99, branchCoverage: 99, functionCoverage: 99 });
+      const stream = run({ files, coverageIncludeGlobs: '**', coverage: true,
```

Comment:
Can you also add a test without `coverageIncludeGlobs`?

Reply:
Sure